hack/api-reference/core.md

```diff
@@ -106,20 +106,6 @@ BackupBucketProvider
 </tr>
 <tr>
 <td>
-<code>providerConfig</code></br>
```

Comment:
Why are there so many changes unrelated to your PR?

Reply:
Had to regenerate the OpenApi Spec